frontend/package.json

```diff
@@ -88,6 +88,7 @@
 		"lucide-react": "0.379.0",
 		"mini-css-extract-plugin": "2.4.5",
 		"papaparse": "5.4.1",
+		"rc-tween-one": "3.0.6",
```

Comment:
the last commit to this library was 2 years ago... doesn't seem like it's actively maintained

Reply:
It's fine. It's as per antd documentation. Working as expected.

Reply:
https://ant.design/components/tag#tag-demo-animation